.github/workflows/package.yml

```diff
@@ -0,0 +1,87 @@
+name: Packages
+on: [push, pull_request]
```

Comment:
This seems to result in the same job being run twice for pull requests created directly on the openzim/libzim repository (rather than external PRs via forks). For example, see https://github.com/openzim/libzim/pull/411/checks?sha=445424d7c13c913d8b0e45fa2338ff96b6ccc78b . Does it work as intended?

Reply:
Yes, we have a problem/challenge here and in other repos. We want to have the run on push and pull... but without running things twice.

Reply:
I didn't really see another way to always run against PRs and on pushes. Is it a major problem to run it twice? It's relatively quick.

Reply:
To me this is a problem yes: waster ot time, HW resources and confusing for users.